src/components/sidebar/index.js

```diff
@@ -19,7 +20,7 @@ export default function selectSidebar(
     'flame-graph': CallTreeSidebar,
     'stack-chart': null,
     'marker-chart': null,
-    'marker-table': null,
+    'marker-table': MarkerSidebar,
```

Comment:
Note: I don't think we'll want to enable it until it provides new data.

Reply:
This was for testing purpose and will be removed before we merge.